docs/features/copy_file.md

```diff
@@ -14,7 +14,7 @@ nginxC, err := GenericContainer(ctx, GenericContainerRequest{
 		Started: true,
 	})
 
-nginxC.CopyFileToContainer(ctx, "./testdata/hello.sh", "/hello_copy.sh", 700)
+nginxC.CopyFileToContainer(ctx, "./testdata/hello.sh", "/hello_copy.sh", 0700)
```

Comment:
I think there is a linter for that provided by golangci but it usually suggest something like that 
```suggestion
nginxC.CopyFileToContainer(ctx, "./testdata/hello.sh", "/hello_copy.sh", 0o700)
```
If you find the linter it would worth adding it to golangci lint config with your PR , don't you think?

Reply:
I'm a newbie in golang, so maybe I'm wrong, but...
It seems golangci-lint doesn't have a check for octals: https://golangci-lint.run/usage/linters/
go-critic has such a check, but go-critic is not used in this repo: https://go-critic.com/overview.html#octalliteral

Do you want me to change the examle code to the suggested 0o... format anyway?